tests/devices/qubit/test_simulate.py

```diff
@@ -177,6 +171,59 @@ def test_tf_results_and_backprop(self):
         assert qml.math.allclose(grad1[0], -tf.sin(phi))
 
 
+class TestSpecialMeasurements:
+    # pylint: disable=too-few-public-methods
+    @pytest.mark.parametrize(
+        "obs, expected",
+        [
+            (
+                qml.Hamiltonian([-0.5, 2], [qml.PauliX(0), qml.PauliX(0) @ qml.PauliX(1)]),
+                -0.25461982,
```

Comment:
These kinds of "magic" numbers make for hard-to-debug tests.

One option is to use analytically known solutions and compare them.

For example, above we are using
```
qs = qml.tape.QuantumScript([qml.RX(phi, wires=0)], [qml.expval(qml.PauliY(0)), qml.expval(qml.PauliZ(0))])
```
and we know that gives `(-sin(phi), cos(phi))`.

Thus we can compare against the Hamiltonian `2.5 * qml.PauliY(0) + -0.15 * qml.PauliZ(0)`.  We can then analytically compare the result to `-2.5*sin(phi) - 0.15*cos(phi)`.